consensus/istanbul/core/testbackend_test.go

```diff
@@ -130,16 +129,15 @@ func (self *testSystemBackend) Verify(proposal istanbul.Proposal, src istanbul.V
 }
 
 func (self *testSystemBackend) Sign(data []byte) ([]byte, error) {
-	testLogger.Warn("not sign any data")
-	return data, nil
+	return self.Address().Bytes(), nil
```

Comment:
Why return the address here?

Reply:
because that's the simplest way to be able to mock out returning the address from `CheckValidatorSignature`

Reply:
Please add a comment explaining why this is done

Reply:
We now use real signing in this test.